types/chartjs-plugin-doughnutlabel-rebourne/index.d.ts

```diff
@@ -0,0 +1,62 @@
+// Type definitions for chartjs-plugin-doughnutlabel-rebourne 3.0
+// Project: https://github.com/alexkuc/chartjs-plugin-doughnutlabel-rebourne
+// Definitions by: Piotr Ładoński <https://github.com/Voodu>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.1
+
+import { Chart, ChartType, Plugin } from 'chart.js';
+import { EmptyObject } from 'chart.js/types/basic';
```

Comment:
```suggestion
import { EmptyObject } from 'chart.js/types/basic';
```
this is only used by internal api, see other comment. Do you need this?